cmd/dra-example-kubeletplugin/driver.go

```diff
@@ -128,7 +138,12 @@ func (d *driver) NodeUnprepareResource(ctx context.Context, req *drapbv1.NodeUnp
 			return fmt.Errorf("error unpreparing devices for claim '%v': %v", req.ClaimUid, err)
 		}
 
-		err = d.nasclient.Update(d.state.GetUpdatedSpec(&d.nascrd.Spec))
+		updatedSpec, err := d.state.GetUpdatedSpec(&d.nascrd.Spec)
+		if err != nil {
+			return err
```

Comment:
we should wrap this error the same as in `NodePrepareResources()`.

Reply:
Line 148 below shoould be wrapped too then ?